templates/zerver/help/mute-a-topic.md

```diff
@@ -19,16 +21,42 @@ They also do not contribute to stream unread counts.
     You can also click on the <i class="fa fa-bell-slash"></i> in the message
     recipient bar to mute the topic.
 
+{tab|mobile}
+
+{!topic-long-press-menu.md!}
+
+1. Tap **Mute topic**.
+
+{!topic-long-press-menu-tip.md!}
+
 {end_tabs}
 
 ## Browse and unmute previously muted topics
 
 {start_tabs}
 
+{tab|desktop-web}
+
 {settings_tab|muted-topics}
 
 1. Review and unmute any muted topics.
 
+{tab|mobile}
+
+1. Tap the streams (<i class="fa fa-hashtag"></i>) tab.
```

Comment:
I wonder if we should format this as **Streams** (<i class="fa fa-hashtag"></i>), similarly to **Recent topics** on https://zulip.com/help/recent-topics?

Reply:
I read that sort of formatting as meaning that the text is a label you can look for in the UI. Here, it's just an icon with no text (unless you're on an iPad, or IIRC if you're using screen-reading software.) So I think leaving it unformatted like this is good.

Reply:
> I think it would be good to decide on:
> 
> * Whether text labels for icon-only buttons should be bold.
> * In which cases to use a capital letter.
> * Whether icons should be labelled based on their shape or function (e.g. ‘gear’ vs. ‘settings’).
> * Whether the icon should appear before the label or after it in parenthesis.

I agree, these are all good questions! And I don't think we currently have consistent answers to any of them.